src/Quaternion.jl

```diff
@@ -47,17 +47,47 @@ julia> quat([1, 2, 3])
  Quaternion{Int64}(3, 0, 0, 0)
 ```
 """
-quat
+function quat end
```

Comment:
Why this change?

Reply:
I was thinking that explicit `funciton quat end` might be better for document generation, but it seems it does not make any difference. I will revert the change later.